tests/integration/rules/always-valid.phpt

```diff
@@ -1,5 +1,3 @@
---TEST--
-alwaysValid()
 --FILE--
 <?php
 require 'vendor/autoload.php';
```

Comment:
This test does not cover anything with exceptions.

Could replace its content with `v::not(v::alwaysValid())->check(...)` and `v::not(v::alwaysValid())->assert(...)`?